src/scripts/menus/app.js

```diff
@@ -38,7 +39,7 @@ const appTemplate = [
 if (isMac) {
     const macAppExtraTemplate = [
         {
-            label: 'Services',
+            label: i18n.__('test'),
```

Comment:
Need to change the test here. Also, should we change all menu options to i18n or does macOs already translate it automatically?

Reply:
Updated - macOs does it automatically!